users/actions.py

```diff
@@ -0,0 +1,64 @@
+EMAIL_TEMPLATE_NAME = 'users/signup_email.html'
+SUBJECT_TEMPLATE_NAME = 'users/signup_email_subject.txt'
+
+
+class SendActivationEmail:
+    def __init__(self, user, site, token_generator):
+        self.user = user
+        self.site = site
+        self.token_generator = token_generator
+
+    def call(self):
+        from django.template import loader
```

Comment:
zgodnie z pep8 importy powinny być na początku pliku: 

> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.

Dodatkowo jeżeli importy są w funkcji to jej wykonanie zajmuje więcej czasu - za każdym wywołaniem musi je wykonać.